STANDARDS.md

```diff
@@ -1,16 +1,66 @@
-# Dataset Standards
 
-This file outlines the standards and requirements needed for submitting a dataset to QCArchive.
+This file outlines the standards and data requirements needed for submitting a dataset to QCArchive.
 This ensures that we have a consistent data model for downstream processes.
 
-## Required fields 
+STANDARDS version: 3-DRAFT (2020-11-13)
+
+We distinguish between standards for the datasets (i.e. the actual data), and the standards for training force fields.
 
-Current list:
-* Ensure all submissions have cmiles, most important are mapped hydrogen smiles
-* Ensure the WBO is requested for all submissions, this should be included in the scf properties list using the flag `wiberg_lowdin_indices`
+# Dataset Standards
 
-## Best practices
-* If any calculations are to be redone from another collection, re-use the old input (coordinates, atom ordering etc) as this will avoid running the calculation again and will just create new references in the database to the old results and should help keep the cost of the calculations down.  
+## Each molecule must have the following information:
+- Canonical isomeric explict hydrogen mapped SMILES
+- Provenence info of SMILES generation (NEW)
+- Coordinates
+- Total Charge
+- Coordinates must be in CMILES order (discuss)
```

Comment:
Yes this should always be true.

Reply:
This is not true.. What I mean here is that our SMILES is nearly always out of order with the 3D, hence why we have the mapped CMILES. We could possibly save headaches by reordering the 3D into the SMILES ordering prioer to submission, such that the mapped SMILES is always :1, :2, :3, :4, etc. We have things in the TK like `from_mapped_smiles`, but this doesn't quite work since the dihedrals index (for TDs) is still in the 3D index system. So, I presented this because, although I don't think it's necessary, it would make using the data much much easier for the general user.

Reply:
Yes, I think this would be the implementation of my note, but the question is... should we do/enforce this? We already have an unmapped version (`canonical_isomeric_explicit_hydrogen_smiles`), which would also mean that the ordering of these atoms is the ordering of the 3d coordinates.

Reply:
Oh I mean the coordinates should always be in the order of the mapped smiles which is always true. This problem is avoided if we always make a molecule from the mapped smiles, this means you can just add a conformer to the molecule without having to do any reordering, the dihedral array in record is then correct and matches the molecule. We could, of course, have QCSubmit enforce canonical ordering of the molecule however to avoid confusion.